Calibration/PPSAlCaRecoProducer/python/ALCARECOPPSCalMaxTracks_cff.py

```diff
@@ -0,0 +1,28 @@
+import FWCore.ParameterSet.Config as cms
+
+import HLTrigger.HLTfilters.hltHighLevel_cfi as hlt
+
+ALCARECOPPSCalMaxTracksFilter = hlt.hltHighLevel.clone(
+  TriggerResultsTag = cms.InputTag("TriggerResults","","HLTX"),
+  eventSetupPathsKey = 'PPSCalMaxTracks',
+)
+
+from EventFilter.CTPPSRawToDigi.ctppsRawToDigi_cff import *
+
+totemRPRawToDigi.rawDataTag = cms.InputTag('hltPPSCalibrationRaw')
+ctppsDiamondRawToDigi.rawDataTag = cms.InputTag('hltPPSCalibrationRaw')
```

Comment:
Also if the parameter exists in the cloned configuration the python type could be dropped.

Reply:
Done in https://github.com/cms-sw/cmssw/pull/36273/commits/18c83a8f88a9ba2168b0438515eb0d211ecdd53c